README.md

```diff
@@ -11,16 +11,10 @@ gitops-promotion interacts with a Git provider to do automatic propagation of co
 
 ## The workflow
 
-gitops-promotion workflow assumes a separation one or more "app" repositories, which results in container images, and the repository (or repositories) that hold manifests that describe how those containers are deployed. We refer to this repository as the "GitOps" repository. Assuming a typical dev/qa/production succession of environments, gitops-promotion is meant to support a workflow that looks like this:
+gitops-promotion workflow assumes a separation of one or more "app" repositories, which results in container images, and the repository (or repositories) that hold manifests that describe how those containers are deployed. We refer to this repository as the "GitOps" repository. Assuming a typical dev/qa/production succession of environments, gitops-promotion is meant to support a workflow that looks like this:
 
 1. A pipeline in `webui` app repository builds, tests and delivers an image to the container registry.
```

Comment:
Wow. I missed the markdown 101. Didn't know you could do lists by only using `1.`. TIL